prayertimes@linworx.org/files/prayertimes@linworx.org/metadata.json

```
@@ -0,0 +1,11 @@
+{
+    "uuid": "prayertimes@linworx.org",
+    "name": "Muslim prayer times",
+    "description": "Muslim daily prayer times desklet",
+    "prevent-decorations": true,
+    "author": "linworx.org",
+    "max-instances": "2",
+    "cinnamon-versions": [
+        "5.8"
+    ]
```

```suggestion
    "cinnamon-versions": [
        "5.8"
    ]
```

I don't think this is needed since the directories are versioned.